### DISCUSSION TITLE
[Feature request] Change domain / merge websites

### DISCUSSION THREAD

**Discussion by xplosionmind** [plausible/analytics#1080] [Anything Else]
If I have a website e.g. `example.net` and I change its domain to `example.io`, I would want to either:
- move all of the visits data I collected up to now from `example.net` to the newly added `example.io` (merging the two websites)
- changing the domain of the website from `example.net` to `example.com`, hence keeping it the same entry on Plausible, but changing its attributes.

**Comment by metmarkosaric** [comment]
instructions on how to change the domain name: https://plausible.io/docs/change-domain-name
